integration_tests/optout.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from '@playwright/test'
+import tokenVerification from './mockApis/tokenVerification'
+import { resetStubs } from './mockApis/wiremock'
+import feComponent from './mockApis/feComponent'
+import assessForEarlyRelease from './mockApis/assessForEarlyRelease'
+import { favicon, getSignInUrl, signIn, signOut, token } from './mockApis/auth'
+
+const baseUrl = 'http://localhost:3007'
+
+async function login({ authorities }: { authorities: string[] }, active = true) {
```

Comment:
**suggestion (non-blocking):** we could possibly move this to somewhere central and share across specs?

Reply:
agreed